napari/_qt/widgets/qt_plugin_sorter.py

```diff
@@ -77,7 +77,25 @@ def __init__(self, item: QListWidgetItem, parent: QWidget = None):
         self.position_label = QLabel()
         self.update_position_label()
 
+        # ~~~show plugin name + function name
+        self.setToolTip("Click and drag to change call order")
         self.plugin_name_label = QLabel(item.hook_implementation.plugin_name)
+        plugin_name_font = QFont()
```

Comment:
I don't think I should create a separate font for this and hardcode the font size, but I'm not sure how to manipulate the theme to do it for me.

Reply:
one option is to use `setObjectName` on a specific object, and then add an entry to one of the qss files like `02_custom.qss` that matches that name.  For instance, lower down in this file, I used:

``python
        title = QLabel('Plugin Sorter')
        title.setObjectName("h3")
```

and then in `02_custom.qss`:
```css
QLabel#h3 {
  font-size: 18px;
  color: {{ secondary }};
}
```

It's fine if you need to do a "one-off" style declaration, but if you can use something that's already being used, all the better (such as `#small_text`)